superset-frontend/src/dashboard/util/getFilterConfigsFromFormdata.js

```diff
@@ -20,89 +20,85 @@
 import { TIME_FILTER_MAP } from '../../visualizations/FilterBox/FilterBox';
 import { TIME_FILTER_LABELS } from '../../explore/constants';
 
-export default function getFilterConfigsFromFormdata(form_data = {}) {
+/**
+ * Parse filters for Table chart. All non-metric columns are considered
+ * filterable values.
+ */
+function getFilterConfigsFromTableChart(form_data = {}) {
+  const { groupby = [], all_columns = [] } = form_data;
+  const configs = { columns: {}, labels: {} };
+  // `groupby` is from GROUP BY mode (aggregations)
+  // `all_columns` is from NOT GROUP BY mode (raw records)
+  const columns = groupby.concat(all_columns);
+  columns.forEach(column => {
+    configs.columns[column] = undefined;
```

Comment:
would `null` be better here? seems like that might match the filterbox empty value? to me that's more an explicit "undefined" versus `undefined`

Reply:
To me, `null` means users specifically choose an empty value for the filter. `undefined` is more like the "data not yet available" default.

In `FilterBox`, the `defaultValue` is also `undefined`, not `null`:
![image](https://user-images.githubusercontent.com/335541/79784053-84373e00-82f6-11ea-8d8c-c4358a9cccf7.png)